apps/web/src/server/lib/nunjucks-filters/url.js

```diff
@@ -27,7 +27,9 @@ export const url = (key, filterArguments) => {
 				step ? `${step}/` : ''
 			}`;
 		case 'view-application':
-			return `${domainUrl}/${domainType || ''}/applications/${applicationId || ''}`;
+			return `${domainUrl}/case/${applicationId ? `${applicationId}/` : ''}${
```

Comment:
better `case 'view-case'` if we stick to CASE for now

Reply:
that case line is existing code not changed in this PR, and it will mean refactoring several pages.  Can we leave this as is?